lerna.json

```
@@ -1,4 +1,9 @@
 {
-  "packages": ["packages/*", "packages/react/examples/*", "axe_core_test"],
+  "packages": [
+    "packages/*",
+    "packages/react/examples/*",
+    "packages/webdriverjs/tests/*",
```

It looks like the are directories other than the examples here. This should likely be:

```suggestion
    "packages/webdriverjs/tests/example",
```